src/browser/components/MainPage.jsx

```diff
@@ -108,6 +121,20 @@ const MainPage = createReactClass({
     ipcRenderer.on('focus-on-webview', () => {
       this.focusOnWebView();
     });
+
+    ipcRenderer.on('protocol-deeplink', (event, lastUrl) => {
+      const mattermostViews = document.getElementsByClassName('mattermostView mattermostView-with-tab');
```

Comment:
This is equal to `props.teams.length`. And you can access to the view via `refs[mattermostView${i}]`. So you don't have to search around DOM.

By adding `getSrc()` member function to `MattermostView`, `MainPage` doesn't have to know about `<webview>`. It should be the role of `MattermostView`.